pkg/vectorize/bin/bin.go

```diff
@@ -87,3 +91,30 @@ func uintToBinary(x uint64) string {
 
 	return string(b[i+1:])
 }
+
+
+func Bin[T constraints.Unsigned | constraints.Signed](intputVector, resultVector *vector.Vector, proc *process.Process) error{
+	xs := vector.MustTCols[T](intputVector)
+	rs := make([]string,0)
+	for idx := range xs {
+		res := uintToBinary(uint64(xs[idx]))
+		rs = append(rs,res)
+	}
+	vector.AppendString(resultVector, rs, proc.Mp())
+	return nil
+}
+
+func BinFloat[T constraints.Float](intputVector, resultVector *vector.Vector, proc *process.Process) error {
+	xs := vector.MustTCols[T](intputVector)
+	rs := make([]string,0)
+	for idx := range xs {
+		val, err := strconv.ParseInt(fmt.Sprintf("%1.0f", xs[idx]), 10, 64)
```

Comment:
Because i want to have an error when the number is too big like 10e50, i saw the oct() function using that way to solve the problem, i had talked it to the PM， he said thats ok.